java/core/src/main/scala/com/azavea/gdal/Native.scala

```diff
@@ -0,0 +1,7 @@
+package com.azavea.gdal
+
+trait Native {
+  protected var nativeHandle = 0l // C++ pointer
```

Comment:
Since JNI bindings is a C interface, we don't have tools to store pointers. So we have to store things manually.